### PR TITLE
Handle prefixed value of -moz-user-select properly

### PR DESCRIPTION
**Pull request by upsuper**
This is supposed to fix [bug 1374996](https://bugzilla.mozilla.org/show_bug.cgi?id=1374996).

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/17568)

### CODE REVIEW

**Comment by upsuper** [comment]
https://treeherder.mozilla.org/#/jobs?repo=try&revision=7b95bbffeb373f28fa25fcc2e68d225a5a2878ae

File: components/style/properties/data.py [hunk 1]
```diff
@@ -75,7 +75,8 @@ def __init__(self, name, values, gecko_constant_prefix=None,
         self.extra_gecko_aliases = parse_aliases(extra_gecko_aliases or "")
         self.extra_servo_aliases = parse_aliases(extra_servo_aliases or "")
         self.consts_map = {} if custom_consts is None else custom_consts
-        self.gecko_strip_moz_prefix = gecko_strip_moz_prefix
+        self.gecko_strip_moz_prefix = True \
```
Comment (nox) [inline components/style/properties/data.py]: Why did you need to change this? You pass `False` in the definition of `-moz-user-select` so why is this needed?

Comment (upsuper) [inline components/style/properties/data.py]: Because when `single_keyword_computed` generates `keyword_kwargs`, it assigns `None` to `gecko_strip_moz_prefix` key if it isn't specified, and `None` is equal to false in condition, which reverses the default value of this parameter.

Comment (Manishearth) [inline components/style/properties/data.py]: Not convinced this is necessary, but r=me

Comment (upsuper) [inline components/style/properties/data.py]: I tried adding `gecko_strip_moz_prefix` to `keyword_kwargs` without changing this, and it failed to build because the moz prefix is incorrectly kept in many other properties, so this is necessary.

Alternatively, we can change how `keyword_kwargs` is generated, use `pop('gecko_strip_moz_prefix', True)` for this instead of just adding it to the list. But having different places maintaining a single default value seems fragile.
